.buildkite/scripts/signAndPublishPackage.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+
+if ! command -v gsutil &> /dev/null ; then
+    echo "⚠️  gsutil is not installed"
+    exit 1
+else
+    echo "✅ gsutil is installed"
+fi
+
+gsutil help
```

Comment:
Is this a script a placeholder for another one that will come later? :thinking:

Reply:
This script should replace the logic to signing a package and publishing it.
Currently, it is here just a placeholder as you mention and it allowed me to check that `gsutil` command is available.